src/core/reader/types/mod.rs

```diff
@@ -81,10 +84,89 @@ impl WasmReadable for VecType {
 /// <https://webassembly.github.io/spec/core/binary/types.html#reference-types>
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub enum RefType {
+    None(ActualRefType),
```

Comment:
Should we have a None type here that takes an ActualRefType?
Something like this is still needed for NULL References at Validation time.

Reply:
Let's talk about this next meeting